plugins/readtimer.koplugin/main.lua

```diff
@@ -15,6 +23,12 @@ local ReadTimer = WidgetContainer:extend{
 }
 
 function ReadTimer:init()
+    self.timer_symbol = "\u{23F2}"  -- ⏲ timer symbol
+    self.timer_letter = "T"
+
+    self.readtimer_show_value_in_footerheader = G_reader_settings:readSetting("readtimer_show_value_in_header")
+    self.readtimer_show_value_in_footer = G_reader_settings:readSetting("readtimer_show_value_in_footer")
```

Comment:
You could have keep the variable short, only the settings (which will be mixed with many others) need the prefix.
` self.show_value_in_footer = G_reader_settings:readSetting("readtimer_show_value_in_footer")`